delayed_job/monitor_delayed_jobs.rb

```diff
@@ -67,22 +70,53 @@ def build_report
           query_hash[key] = query_hash[key].where('queue = ?', option(:queue_name))
         end
       end
+      if option(:exclude_queue_name)
```

Comment:
Nitpick: Considering they both update the same hash key, we could save a line with `elsif`